crashlogs.cpp

```diff
@@ -72,8 +88,11 @@ namespace glaiel::crashlogs {
     //output the crashlog file after a crash has occured
     static void output_crash_log() {
         std::filesystem::path path = get_log_filepath();
-        std::ofstream log(get_log_filepath());
+        const auto output_file = get_log_filepath();
+        std::cout << "Writing crashlog to " << output_file << std::endl;
+        std::ofstream log(output_file);

```

Comment:
no cout, no auto. if we want to allow logging here it should be via a user specifiable callback, I'm not particularly sure its necessary to have on the user side